webapp/components/Table/useTable.js

```diff
@@ -18,6 +18,11 @@ export const useTable = ({ moduleApiUri, module, restParams }) => {
     params: restParams,
   })
 
+  const refetchData = useCallback(() => {
```

Comment:
better initData ?

also at line 22,23 you can invoke this function directly to avoid duplication

Reply:
Yes